gdnative-core/src/string.rs

```diff
@@ -199,15 +199,16 @@ impl_basic_traits!(
     }
 );
 
-impl fmt::Debug for GodotString {
-    fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
-        self.to_string().fmt(f)
+impl fmt::Display for GodotString {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let utf8 = self.to_utf8();
+        f.write_str(&utf8.as_str())
```

Comment:
`Utf8String::as_str` already returns `&str`. Is there a reason to add an explicit borrow here?

Reply:
Ah, thank you! It was left from a previous attempt I didn't like. :D